x-pack/plugin/inference/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/xpack/application/AzureOpenAiServiceUpgradeIT.java

```diff
@@ -49,35 +49,38 @@ public static void shutdown() {
     @AwaitsFix(bugUrl = "Cannot set the URL in the tests")
     public void testOpenAiEmbeddings() throws IOException {
         var openAiEmbeddingsSupported = getOldClusterTestVersion().onOrAfter(OPEN_AI_AZURE_EMBEDDINGS_ADDED);
+        String old_cluster_endpoint_identifier = oldClusterHasFeature("gte_v" + MODELS_RENAMED_TO_ENDPOINTS) ? "endpoints" : "models";
```

Comment:
`gte_` is "Greater Than or Equal to". The `v` is required because the string `MODELS_RENAMED_TO_ENDPOINTS` doesn't contain the `v` in `v8.15.0`. This use of oldClusterHasFeature is documented here https://github.com/elastic/elasticsearch/blob/fa196f72c14cb5379d190f8b7e0439af0ff81bd2/test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestFeatureService.java#L91

I'm happy to add a comment explaining this.

Reply:
A comment would be super helpful, thanks! :)